src/autorest.bicep/src/converter.ts

```diff
@@ -114,7 +114,13 @@ export abstract class SchemaConverter {
             return null;
         }
 
-        const resourceType = new ResourceType(`${fullyQualifiedType}@${result.descriptor.apiVersion}`, result.descriptor.scopeType, result.bodyType);
+        const resourceType = new ResourceType(
+            `${fullyQualifiedType}@${result.descriptor.apiVersion}`,
+            result.descriptor.scopeType,
+            undefined,
```

Comment:
I'm not really sure what `scopeType` means for K8s resources, so no resources have any read-only scopes. To be revisited in a future PR.